app/code/Magento/Catalog/Test/Unit/Model/Product/Option/SaveHandlerTest.php

```diff
@@ -53,8 +54,10 @@ protected function setUp(): void
 
     public function testExecute()
     {
-        $this->optionMock->expects($this->any())->method('getOptionId')->willReturn(5);
-        $this->entity->expects($this->once())->method('getOptions')->willReturn([$this->optionMock]);
+        $this->entity->expects($this->once())->method('getOptionsSaved')->willReturn(false);
+        $this->entity->expects($this->once())->method('getCanSaveCustomOptions')->willReturn(true);
```

Comment:
Is there any reasons to check that method was ran exactly one time? Are those methods slow?

Reply:
So what is the best way to check whether the method was called on not?
Just by calling `$this->any()`?

Reply:
If you don’t need to check how many times it was called - you can use `any`, if you want to check that method was called - `atLeastOnce`, if you want to check if it was called only once or other amount of times - `once` or `exact`, but last case is needed only if this method is quite heavy, for instance doing load of model or runs DB query